people_recognition_2d/src/people_recognition_2d/people_recognizer_2d.py

```diff
@@ -219,6 +221,19 @@ def _shirt_colors_to_label(shirt_colors):
             label += " {}".format(color)
         return label
 
+    @staticmethod
+    def _hair_colors_to_label(hair_colors):
+        """
+        Convert shirt colors array to label string
+
+        :param: shirt_colors: Array to colors
+        :return: string label
```

Comment:
shirt -> hair

Reply:
And I see some significant overlap with `_shirt_colors_to_label`, so maybe reused some code instead of copying

Reply:
Multiple colors?

Reply:
No multiple colors. Currently Person2D has the parameter tshirt color and for the hair color we need to add another parameter?

Reply:
Yes, it is a different property. So we would need to extend the msg for that.